doc/nrf/releases/release-notes-1.4.0.rst

```diff
@@ -0,0 +1,962 @@
+    * Updated functions that end with ``_connect()`` and ``_start()`` to parse complete URLs, with port and schema.
+    * Removed ``port`` field in download_client_config.
+      The port number can now be passed together with the URL.
+    * Removed the ``CONFIG_DOWNLOAD_CLIENT_TLS`` Kconfig option.
+      Now the choice of between secure and non-secure HTTP or CoAP is determined by the schema, or by the security tag if the schema is missing.
+    * No longer uses HTTP range requests when using HTTP, which improves bandwidth.
+    * Parsing of HTTP header fields is now case-insensitive.
+    * Added support for Shell support.
+    * Added ``fragment_siz``e parameter to :c:func:`fota_download_start()` to allow to specify download fragment size at run time.
+
+  * :ref:`lib_fota_download` library:
+
+    * Added a missing call to :c:func:`dfu_target_done` if :c:func:`dfu_target_write` fails.
+    * Added the error cause information to the ``FOTA_DOWNLOAD_EVT_ERROR`` event.
+
+  * :ref:`lte_lc_readme` library:
+
+    * Updated to parse PSM configuration only when the device is registered to a network.
+      This will help avoid confusing error messages.
+    * Added API for setting eDRX Paging Time Window (PTW).
+    * Added support for Release Assistance Indication (RAI).
+    * Added :c:func:`lte_lc_deinit()` function to the API.
+      This function deinitializes the LTE LC module.
+    * Reworked system mode handling as follows:
+
+      * The preferred mode and optionally the fallback mode are now set through Kconfig.
+      * The current mode is the mode read from the device and is changed using :c:func:`lte_lc_system_mode_set()`.
+      * The target mode is the mode that is used when connecting to LTE network, that is when :c:func:`lte_lc_connect()` or  :c:func:`lte_lc_connect_async()` is called.
+        The network is initialized with the configurable (and preferred) system mode.
+        The mode is changed when :c:func:`lte_lc_systm_mode_set()` is called or when connection establishment using preferred mode is unsuccessful and times out.
+
+  * :ref:`liblwm2m_carrier_readme` library:
+
+    * Added the snapshot of the release version 0.10.0.
+      This version is not certified, but can be used for development.
+      See the :ref:`liblwm2m_carrier_changelog` for detailed information.
+
+* :ref:`supl_os_client` library and :ref:`agps_sample` sample:
+
+    * Renamed the sample from nRF Cloud A-GPS.
+    * Added a common interface between A-GPS and SUPL in `nRF Cloud`_.
+    * Added sending of service info after a successful connection to `nRF Cloud`_ has been made.
+
+* :ref:`asset_tracker` application:
+
+    * Added handling of sensor channel ``get`` commands received from `nRF Cloud`_.
+    * Added event handler for :ref:`lte_lc_readme` events.
+    * Added the detection feature when there is no SIM card in the slot.
+    * Added support for Bosch BSEC library 1.4.8.0.
```

Comment:
```suggestion
    * Added support for Bosch BSEC library 1.4.8.0 (see :ref:`zephyr:bme680`).
```
Can't find documentation for the library, but a pointer to the sample might be better than nothing?